src/routes/tekenmethodes/[slug]/stappenplan/+page.svelte

```diff
@@ -39,132 +13,90 @@
 
 <MethodsHeader {data} />
 
-
-<article class="grid">
-  <div class="flex-s">
-    <div class="tags-b">
-
-      <img class="icon" src="/icon1.svg" alt="icon" />
-      <div class="wrapper-methods">
-
-      <img class="icon" src="/images/category.svg" alt="icon" />
-
-      {#each data.methods as method}
-        {#each method.categories as category}
-          <h3 class="tag">
-            {@html category.title}
-          </h3>
+<section class="grid">
+  <div class="tags-container">
+   
+    <div class="tags-group">
+        <img class="icon" src="/images/category.svg" alt="icon" />
+        {#each data.methods as method}
+          {#each method.categories as category}
+            <p class="tag">{@html category.title}</p>
+          {/each}
```

Comment:
Ik denk dat de HTML voor de tags groups semantischer kan; een ul met nested ul's (voor categorieën en materialen) of p elementen voor (tijdsduur en auteur). 

Je zou het eens kunnen checken hoe een screen reader de tags groeps voorleest zoals het nu is, en als je het in een lijst structuur zet

Reply:
check